### PR TITLE
[extension_google_sign_in_as_googleapis_auth] Update example...

### PR DESCRIPTION
**Pull request by kevmoo**
...to use scope from the People API
Remove x bit from readme and pubspec.yaml
Add link to root repo README
## Pre-launch Checklist

- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy].
- [ ] I updated `CHANGELOG.md` to add a description of the change.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [ ] All existing and new tests are passing.

### CODE REVIEW

File: packages/extension_google_sign_in_as_googleapis_auth/example/lib/main.dart [hunk 1]
```diff
@@ -11,10 +11,7 @@ import 'package:extension_google_sign_in_as_googleapis_auth/extension_google_sig
 import 'package:googleapis/people/v1.dart';
 
 GoogleSignIn _googleSignIn = GoogleSignIn(
-  scopes: <String>[
-    'email',
```
Comment (kevmoo) [inline packages/extension_google_sign_in_as_googleapis_auth/example/lib/main.dart:15]: email doesn't seem to be needed...

Comment (ditman) [reply packages/extension_google_sign_in_as_googleapis_auth/example/lib/main.dart:15]: I thought both 'profile' and 'email' were required to sign in, but not seeing profile there is suspicious :P let me run this example real quick!

https://developers.google.com/identity/protocols/oauth2/scopes#google-sign-in

Comment (ditman) [reply packages/extension_google_sign_in_as_googleapis_auth/example/lib/main.dart:15]: ('profile', 'email' and 'openid' seem to get added by the google sign-in API automatically if absent from the original request, good!)

Comment (kevmoo) [reply packages/extension_google_sign_in_as_googleapis_auth/example/lib/main.dart:15]: I've been running it all weekend – no issues!

**Comment by ditman** [comment]
@googlebot I consent. And I love you too.

**Review by ditman — APPROVED** [review]
Let's go! Thanks for the fixes @kevmoo!

**Comment by ditman** [comment]
> @ditman – am I okay to land this?

@kevmoo I've just clicked the button, you're always welcome to merge :P

**Comment by ditman** [comment]
The latest changes have been published to pub.dev as 2.0.2: https://pub.dev/packages/extension_google_sign_in_as_googleapis_auth/changelog#202

**Pull request merged**